pyanaconda/payload/__init__.py

```diff
@@ -145,31 +156,31 @@ def __init__(self, data):
         self._session = requests_session()
 
     def setup(self, storage, instClass):
-        """ Do any payload-specific setup. """
+        """Do any payload-specific setup."""
         self.storage = storage
         self.instclass = instClass
         self.verbose_errors = []
 
     def unsetup(self):
-        """ Invalidate a previously setup paylaod. """
+        """Invalidate a previously setup payload."""
         self.storage = None
         self.instclass = None
 
     def preStorage(self):
-        """ Do any payload-specific work necessary before writing the storage
-            configuration.  This method need not be provided by all payloads.
+        """Do any payload-specific work necessary before writing the storage
+        configuration.  This method need not be provided by all payloads.
```

Comment:
This is on purpose. I don't know exactly why clumes is using double space everywhere.